docs/contracts/v2/reference/05-access-control.md

```diff
@@ -4,32 +4,34 @@ sidebar_position: 5
 title: "Access Control"
 ---
 
-With the exception of the [admin functions](/docs/concepts/05-governance.md), all functionality in Sablier is related to
-one or more streams.
+With the exception of the [allow list](/contracts/v2/reference/core/abstracts/abstract.SablierV2Lockup#allowtohook), all
```

Comment:
As per my other comment, the Governance document should be reinstantiated and this link should be updated accordingly.

Reply:
https://github.com/sablier-labs/docs/pull/138/commits/debe1e613bf523b1f55a330acd3ce247cf09d196